app/views/smart_proxies/index.html.erb

```diff
@@ -7,8 +7,13 @@
   <thead>
     <tr>
       <th class="col-md-2"><%= s_("SmartProxy|Name") %></th>
-      <th class="col-md-4"><%= s_("SmartProxy|Url") %></th>
-      <th class="col-md-4"><%= _("Features") %></th>
+      <% if SETTINGS[:locations_enabled] %>
+        <th class="col-md-2 hidden-sm hidden-xs nbsp"><%= _('Locations') %></th>
+      <% end %>
+      <% if SETTINGS[:organizations_enabled] %>
+        <th class="col-md-2 hidden-sm hidden-xs nbsp"><%= _('Organizations') %></th>
+      <% end %>
+      <th class="col-md-4"><%= _('Features') %></th>
       <th class="col-md-1"><%= _('Status') %></th>
       <th><%= _('Actions') %></th>
```

Comment:
Please add a width to the action column so it doesn't expand too much when locs/orgs are disabled